config/default/wiki.yaml

```diff
@@ -5,19 +5,19 @@ ingress:
     "cert-manager.io/cluster-issuer": "letsencrypt-prod"
     "nginx.ingress.kubernetes.io/ssl-redirect": "true"
   hosts:
-    - host: wiki-test.jenkins.io
```

Comment:
would be good to have some home page content before making this live I think:

https://wiki-test.jenkins.io

also the 404 page is referring to freenode irc

Reply:
> would be good to have some home page content before making this live I think:

Agreed.

> also the 404 page is referring to freenode irc

Not sure I follow, do you have an example?

Reply:
Previously the home page was returning a 404 and there's a butler page that says to contact jenkins-infra on freenode IRC